src/mudlet.cpp

```diff
@@ -672,242 +376,613 @@ mudlet::mudlet()
-#if defined(Q_OS_MACOS)
-    triggersKeySequence = QKeySequence(Qt::CTRL | Qt::Key_E);
-    showMapKeySequence = QKeySequence(Qt::CTRL | Qt::Key_M);
-    inputLineKeySequence = QKeySequence(Qt::CTRL | Qt::Key_L);
-    optionsKeySequence = QKeySequence(Qt::CTRL | Qt::Key_P);
-    notepadKeySequence = QKeySequence(Qt::CTRL | Qt::Key_N);
-    packagesKeySequence = QKeySequence(Qt::CTRL | Qt::Key_O);
-    modulesKeySequence = QKeySequence(Qt::CTRL | Qt::Key_I);
-    multiViewKeySequence = QKeySequence(Qt::CTRL | Qt::ALT | Qt::Key_V);
-    connectKeySequence = QKeySequence(Qt::CTRL | Qt::ALT | Qt::Key_C);
-    disconnectKeySequence = QKeySequence(Qt::CTRL | Qt::Key_D);
-    reconnectKeySequence = QKeySequence(Qt::CTRL | Qt::Key_R);
-#else
-    triggersKeySequence = QKeySequence(Qt::ALT | Qt::Key_E);
-    showMapKeySequence = QKeySequence(Qt::ALT | Qt::Key_M);
-    inputLineKeySequence = QKeySequence(Qt::ALT | Qt::Key_L);
-    optionsKeySequence = QKeySequence(Qt::ALT | Qt::Key_P);
-    notepadKeySequence = QKeySequence(Qt::ALT | Qt::Key_N);
-    packagesKeySequence = QKeySequence(Qt::ALT | Qt::Key_O);
-    modulesKeySequence = QKeySequence(Qt::ALT | Qt::Key_I);
-    multiViewKeySequence = QKeySequence(Qt::ALT | Qt::Key_V);
-    connectKeySequence = QKeySequence(Qt::ALT | Qt::Key_C);
-    disconnectKeySequence = QKeySequence(Qt::ALT | Qt::Key_D);
-    reconnectKeySequence = QKeySequence(Qt::ALT | Qt::Key_R);
-#endif
-    connect(this, &mudlet::signal_menuBarVisibilityChanged, this, &mudlet::slot_update_shortcuts);
-
-    mpSettings = getQSettings();
-    readLateSettings(*mpSettings);
-    // The previous line will set an option used in the slot method:
-    connect(mpMainToolBar, &QToolBar::visibilityChanged, this, &mudlet::slot_handleToolbarVisibilityChanged);
-
-#if defined(INCLUDE_UPDATER)
-    updater = new Updater(this, mpSettings);
-    connect(updater, &Updater::signal_updateAvailable, this, &mudlet::slot_updateAvailable);
-    connect(dactionUpdate, &QAction::triggered, this, &mudlet::slot_check_manual_update);
-#if defined(Q_OS_MACOS)
-    // ensure that 'Check for updates' is under the Applications menu per convention
-    dactionUpdate->setMenuRole(QAction::ApplicationSpecificRole);
-#else
-    connect(updater, &Updater::signal_updateInstalled, this, &mudlet::slot_update_installed);
-#endif // !Q_OS_MACOS
-#endif // INCLUDE_UPDATER
 
-    if (!mToolbarIconSize) {
-        setToolBarIconSize(mEnableFullScreenMode ? 2 : 3);
+    for (auto& translationFileName : qAsConst(translationFilesList)) {
+        QString languageCode(translationFileName);
+        languageCode.remove(QStringLiteral("mudlet_"), Qt::CaseInsensitive);
+        languageCode.remove(QStringLiteral(".qm"), Qt::CaseInsensitive);
+        int percentageTranslated = -1;
+
+        auto pMudletTranslator = new QTranslator();
+        auto translationPathFileName(path % QLatin1Char('/') % translationFileName);
```

Comment:
```suggestion
        auto translationFilePath(path % QLatin1Char('/') % translationFileName);
```

Reply:
When bandying `QString`s around that may be paths and or file names I like to use PathFile when it is something that has *both* of those parts and Name because it is a text/QString - so i can avoid mixing it up with a `QDir` or `QFile`/`QFileInfo` entity...

... so this is telling me that it is the name of a translation *file* and is both the path **and** file name.